src/main/scala/io/iohk/ethereum/domain/Transaction.scala

```diff
@@ -6,9 +6,9 @@ import org.spongycastle.util.encoders.Hex
 
 object Transaction {
 
-  val NonceLength = 32
-  val GasLength = 32
-  val ValueLength = 32
+  val NonceLength = 33
+  val GasLength = 33
+  val ValueLength = 33
```

Comment:
As the way this are checked in `SignedTransactionValidator` was fixed, should we return this lengths to `32` bytes?